15-Swift/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# Build stage
+FROM swift:latest as builder
```

Comment:
why debian?

Reply:
slim tags for this image are not design to build project they can just run the code,there isno other specific tag to select

Reply:
let me check it later ❤️